src/vos/tests/vts_aggregate.c

```diff
@@ -354,7 +354,7 @@ aggregate_basic(struct io_test_args *arg, struct agg_tst_dataset *ds,
 	assert_non_null(buf_u);
 
 	for (epoch = epr_u->epr_lo; epoch <= epr_u->epr_hi; epoch++) {
-		if (punch_nr > 0 && punch_epoch[punch_idx] == epoch) {
+		if (punch_idx < punch_nr && punch_epoch[punch_idx] == epoch) {
```

Comment:
I thought I made this change in a recent patch.

Reply:
it was a while ago.

Reply:
ah, just downloaded your patch...looks like the only difference is the assert on 359 is removed in master as it's redundant....Not sure why github marks that line as the change.